src/components/RepoList.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div id="repoList">
```

Comment:
```suggestion
  <article id="repoList">
```
사실 vue 알 못 이라. 
하이라키가 명확히 그려지진 않아, 적절한 semantic tag 부여인진 모르겠는데,

PC 페이지로 만들어지는 만큼 semantic tag들을 챙겨주시면 더 좋을 것 같습니다.

Reply:
그러네요! 개발하면서 시멘틱태그를 미쳐 신경쓰지 못했네요 감사합니다!